md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -22,14 +22,23 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
 :::
 
 **JRE requirements:**
-Versions 7.0 to 7.4 only support JRE version 7. If you comes from older versions of Bonita supporting JRE 6, you must also upgrade your JRE to version 7.
-Version 7.5 only supports JRE version 8. If migrating to version 7.5, please upgrade your JRE to version 8.
-For more info, see Support Guide and Supported Environment Matrix for Server.
+* Versions 7.0 to 7.4 only support JRE version 7. If you comes from older versions of Bonita supporting JRE 6, you must also upgrade your JRE to version 7.
+* Version 7.5 only supports JRE version 8. If migrating to version 7.5, please upgrade your JRE to version 8.
+* For more info, see Support Guide and Supported Environment Matrix for Server.
 
-The tool migrates your platform (_bonita_home_ folder and the database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating. If you are running a
-Bonita Subscription Pack edition, you need a valid license for your target version.
-If you are upgrading to a new maintenance version and not changing the minor version number (for example, you are migrating from 6.3.0 to 6.3.1),
-your current license remains valid after migration.
+::: warning
+**<a id="rdbms_requirements" />RDBMS requirements:**
+The version targeted may not support the version of the database that is being migrated. You may then need to upgrade the version of your database prior to running the migration tool.
+* Please check the [database requirements](hardware-and-software-requirements.md).
+* If you need to upgrade your database:
+   * Please make sure to apply all the [RDBMS customisations required by Bonita](database-configuration.md#toc5) when setting up the new version.
+   * Please make sure to use the [appropriate JDBC driver](database-configuration.md#toc3)
+ :::
+
+**Community/Subscription edition:**
+The tool migrates your platform (_bonita_home_ folder and the database). You cannot [change edition](upgrade-from-community-to-a-subscription-edition.md) while migrating.
+* If you are running a Bonita Subscription Pack edition, you need a valid license for your target version.
```

Comment:
```suggestion
* If you are running a Bonita Subscription edition, you need a valid license for your target version.
```

Reply:
Same answer as the one to Thomas' comment.

I'm fine to remove the 'Pack' if that is what Product recommends.